src/StudentDashboard/MoodForm.scss

```diff
@@ -0,0 +1,20 @@
+.student-mood-input {
+  height: 15vh;
+
+}
+
+.submit-mood-button {
+  margin-top: 15vh;
+  height: 5vh;
+  width: 15vw;
+  background-color: #CC79A7;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+  border: 4px ridge #0072B2
+}
+h4 {
```

Comment:
fyi I believe that h4 style will style ALL h4s in the app, not just this component.  You could nest this or just give it a class name.

Reply:
Ah thanks for noticing that